pkg/release/build.go

```diff
@@ -238,6 +244,8 @@ type Platform int
 const (
 	// PlatformLinux is the Linux x86_64 target.
 	PlatformLinux Platform = iota
+	// PlatformLinuxFIPS is the Linux x86_64 target built with FIPS support.
```

Comment:
Nit: Do we want FIPS + ARM?

Reply:
Yes. I just started with one platform for now, because I need to fix the header search issue when we cross build arm64 on amd64.

Reply:
We decided to go with intel only for now.